### PR TITLE
BZ 1591637 - Add info about mode permissions

### PR DESCRIPTION
**Pull request by ghost**
https://bugzilla.redhat.com/show_bug.cgi?id=1591637

### CODE REVIEW

**Comment by weherdh** [comment]
Looks good to me, thanks.
/lgtm

**Comment by tmorriso-rh** [comment]
@openshift/team-documentation PTAL.

File: install_config/persistent_storage/persistent_storage_azure_file.adoc [hunk 1]
```diff
@@ -42,24 +42,73 @@ $ /usr/sbin/setsebool -P virt_use_samba on
 $ /usr/sbin/setsebool -P virt_sandbox_use_samba on
 ----
 
+. Type the `mount` command to check `dir_mode` and `file_mode` permissions, for example:
```
Comment (kalexand-rh) [inline install_config/persistent_storage/persistent_storage_azure_file.adoc]: s/Type/Run

File: install_config/persistent_storage/persistent_storage_azure_file.adoc [hunk 2]
```diff
@@ -42,24 +42,73 @@ $ /usr/sbin/setsebool -P virt_use_samba on
...
++
+----
+$ mount
+----
+
+If the `dir_mode` and `file_mode` permissions are set to `0755`,  change the
+default value `0755` to `0777` (or `0775`). This manual step is required because
```
Comment (kalexand-rh) [inline install_config/persistent_storage/persistent_storage_azure_file.adoc]: Do we have an opinion on one or the other, or do we expect the user to know the difference?

Remove the parentheses.

File: install_config/persistent_storage/persistent_storage_azure_file.adoc [hunk 3]
```diff
@@ -42,24 +42,73 @@ $ /usr/sbin/setsebool -P virt_use_samba on
...
+the default `dir_mode` and `file_mode` permissions changed from `0777` to `0755`
+in {product-title} 3.9. The following examples show configuration files with the 
+changed values.
+
+.Example persistent volume configuration file
+----
+# cat azf-pv.yml
```
Comment (kalexand-rh) [inline install_config/persistent_storage/persistent_storage_azure_file.adoc]: I'd remove the cat commands and change the source type to yaml. (It doesn't look like you use these filenames in this topic.)

File: install_config/persistent_storage/persistent_storage_azure_file.adoc [hunk 4]
```diff
@@ -42,24 +42,73 @@ $ /usr/sbin/setsebool -P virt_use_samba on
...
+  - dir_mode=0777
+  - file_mode=0777
+parameters:
+  storageAccount: ocp39str
+  location: centralus
+----
+
 [[azure-file-regional-cloud]]
-== Configuring Azure file for regional cloud
+== Configuring Azure File for regional cloud
```
Comment (kalexand-rh) [inline install_config/persistent_storage/persistent_storage_azure_file.adoc]: I'd update the H1 to sentence case and update the topics.yml file.

**Comment by gaurav-nelson** [comment]
@tmorriso-rh Any reason theses changes are not not made to master and 3.10 ?

Looks like this should be in both branches.

**Pull request merged**